static/src/javascripts/projects/common/modules/navigation/membership.js

```diff
@@ -0,0 +1,31 @@
+define(['common/modules/commercial/user-features', 'common/utils/fastdom-promise', 'Promise', 'common/utils/$'], function (userFeatures, fastdom, Promise, $) {
+    var LAST_CLASS = 'brand-bar__item--split--last';
+
+    function init() {
+        var refreshUser = new Promise(function (resolve) {
```

Comment:
there's no need to assign this `Promise` to a var really, I'd return the entire chain from the function instead